From e03eb61bf678a19c4d24abce4c9984f9f2684e35 Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Mon, 15 Aug 2016 20:23:47 -0700
Subject: [PATCH 1947/2748] HDFS-10747. o.a.h.hdfs.tools.DebugAdmin usage
 message is misleading. (Contributed by Mingliang
 Liu)

(cherry picked from commit e9eaad0e677f38018572db4a3e9adc9f4e8519a5)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HDFSCommands.md

Change-Id: I8386cb357b909b97c497f6565222c943692f3841
---
 .../org/apache/hadoop/hdfs/tools/DebugAdmin.java   |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DebugAdmin.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DebugAdmin.java
index 41f1ca4..7db179d 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DebugAdmin.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DebugAdmin.java
@@ -85,7 +85,7 @@
   private class VerifyBlockChecksumCommand extends DebugCommand {
     VerifyBlockChecksumCommand() {
       super("verify",
-"verify [-meta <metadata-file>] [-block <block-file>]",
+"verify -meta <metadata-file> [-block <block-file>]",
 "  Verify HDFS metadata and block files.  If a block file is specified, we\n" +
 "  will verify that the checksums in the metadata file match the block\n" +
 "  file.");
@@ -199,7 +199,7 @@ int run(List<String> args) throws IOException {
   private class RecoverLeaseCommand extends DebugCommand {
     RecoverLeaseCommand() {
       super("recoverLease",
-"recoverLease [-path <path>] [-retries <num-retries>]",
+"recoverLease -path <path> [-retries <num-retries>]",
 "  Recover the lease on the specified path.  The path must reside on an\n" +
 "  HDFS filesystem.  The default number of retries is 1.");
     }
-- 
1.7.9.5

